feature/amazon/src/main/java/com/revenuecat/purchases/amazon/AmazonBilling.kt

```diff
@@ -430,7 +430,7 @@ internal class AmazonBilling constructor(
              * since there's no terms and we can just use the sku
              */
             val amazonPurchaseWrapper = receipt.toStoreTransaction(
-                sku = storeProduct.sku,
+                productId = storeProduct.sku,
```

Comment:
Should we also deprecate `sku` in the `StoreProduct`?

Reply:
yeah definitely! i can get that in this PR too

Reply:
it ended up touching a ton of files, so i opened a separate PR: https://github.com/RevenueCat/purchases-android/pull/680